s5_base/run.sh

```diff
@@ -98,21 +103,31 @@ if [ $stage -le 4 ]; then
         echo "Create the lexicon"
         [ -d $localdict ] && rm -r $localdict
         mkdir -p $localdict data/lang/log
-        $train_cmd data/lang/log/prep_lang.log local/prep_lang.sh $prondict $localdict data/lang
+        $train_cmd data/lang/log/prep_lang.log \
+        local/prep_lang.sh \
+        $prondict $localdict data/lang
     fi
 fi
 
 if [ $stage -le 5 ]; then
     echo "Preparing a pruned trigram language model"
     mkdir -p data/log
-    $train_cmd --mem 24G data/log/make_LM_3g.log local/make_LM.sh --order 3 --carpa false \
-      --min1cnt 20 --min2cnt 10 --min3cnt 2 $lm_train data/lang $localdict/lexicon.txt data \
-      || error 1 "Failed creating a pruned trigram language model";
-
+    $train_cmd --mem 24G data/log/make_LM_3g.log \
```

Comment:
Shouldn't the mem parameter be located in cmd.sh?

Reply:
The problem with these is that the memory varies with how big the input is so I would then have to create two new commands in the cmd.sh and I wasn't sure whether I should do that